client/src/FooterForm.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+
+export class FooterForm extends Component {
```

Comment:
If you aren't going to use this form here and instead use the form in your contact me component, you can remove this form and then link to your contact me page so that you don't have a form that doesn't do anything. Or you could pass down your email handlers to your FooterForm component and use those to be able to send mail from this form as well.